crates/driver/src/tests/setup/mod.rs

```diff
@@ -645,7 +645,7 @@ impl Setup {
     }
 
     fn deadline(&self) -> chrono::DateTime<chrono::Utc> {
-        time::now() + chrono::Duration::seconds(2)
+        time::now() + chrono::Duration::milliseconds(5500)
```

Comment:
Why do we still need to change this value in the test?

Reply:
We should be fine without the change, but the tests will fail with the next PR where we respect the deadlines... reverted, so let's make this change in the PR that respect deadlines.